hack/lib/tracing.bash

```diff
@@ -90,7 +92,7 @@ spec:
           limits:
             memory: 10Gi
           requests:
-            memory: 256Mi
+            memory: ${ZIPKIN_MEMORY_REQUESTS}
```

Comment:
should we define a default for ZIPKIN_MEMORY_REQUESTS in install_zipkin_tracing , in case it is invoked on its own, and not inside `install_tracing`

Reply:
OK. Pushed.